charts/spire/charts/spire-server/templates/tornjak-config.yaml

```diff
@@ -42,5 +42,13 @@ data:
         }
       }
     {{- end }}
+    {{- if ne .Values.tornjak.config.userManagement.issuer "" }}
+      UserManagement "KeycloakAuth" {
+        plugin_data {
+          issuer = "{{ .Values.tornjak.config.userManagement.issuer }}"
```

Comment:
Is the issuer being used to look up the jwk's too? If so, how is it finding the right url? Is there a missing option here?